apps/website/src/components/logo.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import { match, P } from 'ts-pattern'
+
+import logoSrc from '../../public/images/logo/default.svg'
+import logoDevSrc from '../../public/images/logo/dev.svg'
+import logoLearnSrc from '../../public/images/logo/learn.svg'
+
+type Props = {
+  label?: boolean
+}
+
+export const Logo = (props: Props) => {
+  const { label = true } = props
```

Comment:
Just my POV, since it's just boolean and you set default to `true`, you have to pass prop `label={false}`. I'd rather use opposite in this case therefore `noLabel`, default to `false` and if dev doesn't want to use label, they just pass `noLabel` prop. Wdyt?